cas/softcas/softcas.go

```diff
@@ -112,6 +113,17 @@ func (c *SoftCAS) RevokeCertificate(req *apiv1.RevokeCertificateRequest) (*apiv1
 	}, nil
 }
 
+// CreateCRL will create a new CRL based on the RevocationList passed to it
+func (c *SoftCAS) CreateCRL(req *apiv1.CreateCRLRequest) (*apiv1.CreateCRLResponse, error) {
+
+	revocationListBytes, err := x509.CreateRevocationList(rand.Reader, req.RevocationList, c.CertificateChain[0], c.Signer)
```

Comment:
The signer and certificate chain should not be used here. The last version in the `master` branch has a method that returns the right one:
https://github.com/smallstep/certificates/blob/abf5fc32a3417ca95800bf1d4c8cef2030cf8782/cas/softcas/softcas.go#L210-L216